### ISSUE TITLE
Doesn't work with background urls

### ISSUE THREAD

**Issue by brokenalarms** [timhudson/css-styles#1]
incorrectly splits at every single ':', ignoring when they are a valid part of a property.
`background: url(https://newevolutiondesigns.com/images/freebies/city-wallpaper-37.jpg) center cover"`

returns
`background: url(https`.

**Comment by brokenalarms** [comment]
You want

```javascript
function cssStyleParse(string) {
  if (!string) return {}

  return string.split(';').reduce(function (memo, declaration) {
    if (!declaration) return memo
    var keyEndIndex = declaration.indexOf(':');
    const key = declaration.substring(0, keyEndIndex);
    const prop = declaration.substring(keyEndIndex +1).trim();
    memo[camelcase(key)] = prop
    return memo
  }, {})
}
```

**Comment by brokenalarms** [comment]
Thanks for turnaround!

**Comment by timhudson** [comment]
Thank you @brokenalarms! Published in `1.0.2`
